celery/concurrency/gevent.py

```diff
@@ -16,15 +17,22 @@
 # We cache globals and attribute lookups, so disable this warning.
 
 
+def apply_target(target, args=(), kwargs=None, callback=None,
+                 accept_callback=None, getpid=None, **_):
+    kwargs = {} if not kwargs else kwargs
+    return base.apply_target(target, args, kwargs, callback, accept_callback,
+                             pid=getpid(), **_)
+
+
 def apply_timeout(target, args=(), kwargs=None, callback=None,
-                  accept_callback=None, pid=None, timeout=None,
+                  accept_callback=None, getpid=None, timeout=None,
```

Comment:
this will not introduce any breaking change, right?

Reply:
apply_timeout called in TaskPool.on_apply, when set timeout use apply_timeout otherwise apply_target.
This modification keeps the parameters of the two functions consistent.
Find sage did not find the external call.